systemd.sh

```diff
@@ -0,0 +1,9 @@
+package: systemd
+version: "237"
+system_requirement_missing: |
+   Please install systemd develpment package on your system.
+   If they are there, make sure you have them in your default path or check you have `which` installed.
+system_requirement: ".*"
```

Comment:
At least `(?!osx)` since systemd does not even exist there. We should avoid the dependency completely.

Reply:
Done, with the fix for the modulefile.